src/main/java/com/incognia/api/IncogniaAPI.java

```diff
@@ -35,7 +35,7 @@
 public class IncogniaAPI {
   private static final String API_URL = "https://api.incognia.com";
   private static final String EVALUATION_PARAMETER = "eval";
-
+  private static final String DRY_RUN_PARAMETER = "dry_run";
   private final TokenAwareNetworkingClient tokenAwareNetworkingClient;
```

Comment:
nit: Keep this line that separates the more qualified attributes from the TokenAwareNetworkingClient that is less qualified.